deploy/kubedirector/kubedirector.hpe.com_kubedirectorapps_crd.yaml

```diff
@@ -180,6 +185,11 @@ spec:
               items:
                 type: string
                 pattern: '^/.*[^/]$'
+            defaultEventList:
+              type: array
+              items:
+                type: string
+                pattern: '^[a-z0-9]([-a-z0-9]*[a-z0-9])?$'
```

Comment:
similar comment about enumeration

Reply:
Also of course, once this PR is merged we then need to look at how to express this as v1beta2 rather than just modifying the already-released v1beta1 ... we have been talking about this offline but I wanted to record it here too.

https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/

Reply:
Changed this. Kartik and I will come up with a plan for the v1beta2 plan.